### ISSUE TITLE
Python SDK - Dataset.iterate_items() TypeError on Apify

### ISSUE THREAD

**Issue by lucaromagnoli** [apify/apify-sdk-python#320]
Hello, 

I'm getting the following error when doing an async iteration over `Dataset.iterate_items()` using Python SDK. 
This error only happens when running on Apify infrastructure, not when I run it locally.

e.g.
```Python
async for item in urls_dataset.iterate_items():
    url = item.get("url")
```

Error:
`TypeError: 'async for' requires an object with __aiter__ method, got coroutine`

**Comment by janbuchar** [comment]
Hello, thanks for your interest in Apify and for reporting this! This is pretty obviously a bug, we'll fix it as soon as possible

**Comment by Pijukatel** [comment]
Fix will probably be included in apify 2.0.2
